Solutions/Darktrace/Analytic Rules/CreateAlertFromSystemStatus.yaml

```diff
@@ -0,0 +1,33 @@
+id: 2e629769-60eb-4a14-8bfc-bde9be66ebeb
+kind: scheduled
```

Comment:
Kind should be "Scheduled" to "scheduled"

Reply:
Capitalisation changed.